Doc/library/asyncio-task.rst

```diff
@@ -602,6 +602,84 @@ Waiting Primitives
            # ...
 
 
+Running in Threads
+==================
+
+.. coroutinefunction:: to_thread(func, /, \*args, \*\*kwargs)
+
+   Asynchronously run function *func* in a separate thread.
```

Comment:
I would also document that the `*args` and `**kwargs` are passed to `func` (as opposed to being reserved for modifying `to_thread()`'s behavior), and also in the docstring below.

Reply:
That's a good idea, thanks.